Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,13 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  const reducer = (accumulator, currentValue) => {
+    if (!accumulator.includes(currentValue)) {
+      accumulator.push(currentValue);
+    }
+    return accumulator;
+  };
+  return arr.reduce(reducer, []);
 }
```

Comment:
There are multiple ways of doing this but this is a perfectly acceptable solution.